pkg/manager/manager.go

```diff
@@ -304,6 +332,19 @@ func defaultNewClient(cache cache.Cache, config *rest.Config, options client.Opt
 	}, nil
 }
 
+// defaultHealthProbeListener creates the default health probes listener bound to the given address
+func defaultHealthProbeListener(addr string) (net.Listener, error) {
+	if addr == "" || addr == "0" {
```

Comment:
Did we intend for health probes to not be served by default? By my reading, since `Options.HealthProbeBindAddress` has no default value set apart from the empty string, we get no probe server unless we set it explicitly.